src/issues/util.ts

```diff
@@ -357,7 +357,7 @@ export async function createGithubPermalink(gitAPI: GitAPI, positionInfo?: NewIs
 		return { permalink: undefined, error: 'There is no suitable remote.' };
 	}
 	const pathSegment = document.uri.path.substring(repository.rootUri.path.length);
-	const expr = /^((git\@github\.com\:)|(https:\/\/github\.com\/))(.+\/.+)\.git$/;
+	const expr = /^((git\@github\.com\:)|(https:\/\/github\.com\/))(.+\/.+)(\.git)?$/;

```

Comment:
I think it would be good to reuse the code we have for parsing github urls, `new Protocol(upstream.fetchUrl)`, it has `repositoryName` and `owner` properties

Reply:
I had forgotten about Protocol, thanks!